packages/ignite-basic-generators/templates/examples/GridExample.js.ejs

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React, { PropTypes } from 'react'
+import { View, Text, ListView } from 'react-native'
+import { connect } from 'react-redux'
+// import { Actions as NavigationActions } from 'react-native-router-flux'
+
+// For empty lists
+// import AlertMessage from '../Components/AlertMessage'
+
+// Styles
+import styles from './Styles/GridExampleStyle'
+
+class GridExample extends React.Component {
+
+  state: {
+    dataSource: Object
+  }
+
+  constructor (props) {
+    super(props)
+    // If you need scroll to bottom, consider http://bit.ly/2bMQ2BZ
+
+    /* ***********************************************************
+    * STEP 1
+    * This is an array of objects with the properties you desire
+    * Usually this should come from Redux mapStateToProps
+    *************************************************************/
+    const dataObjects = [
+      {title: 'First Title', description: 'First Description'},
+      {title: 'Second Title', description: 'Second Description'},
+      {title: 'Third Title', description: 'Third Description'},
+      {title: 'Fourth Title', description: 'Fourth Description'},
+      {title: 'Fifth Title', description: 'Fifth Description'},
+      {title: 'Sixth Title', description: 'Sixth Description'},
+      {title: 'Seventh Title', description: 'Seventh Description'}
+    ]
+
+    /* ***********************************************************
+    * STEP 2
+    * Teach datasource how to detect if rows are different
+    * Make this function fast!  Perhaps something like:
+    *   (r1, r2) => r1.id !== r2.id}
+    *************************************************************/
+    const rowHasChanged = (r1, r2) => r1 !== r2
+
+    // DataSource configured
+    const ds = new ListView.DataSource({rowHasChanged})
+
+    // Datasource is always in state
+    this.state = {
+      dataSource: ds.cloneWithRows(dataObjects)
+    }
+  }
+
+  /* ***********************************************************
+  * STEP 3
+  * `_renderRow` function -How each cell/row should be rendered
+  * It's our best practice to place a single component here:
+  *
+  * e.g.
+    return <MyCustomCell title={rowData.title} description={rowData.description} />
+  *************************************************************/
+  _renderRow (rowData) {
```

Comment:
Not a fan of underscore functions.  I know these are old example files you're bringing in though.